src/models/msg.js

```diff
@@ -40,6 +40,7 @@ class Msg {
 
 	isLoggable() {
 		return this.type !== Msg.Type.MOTD &&
```

Comment:
I didn't realize we didn't log motds, maybe we should?

Reply:
That would be pretty spammy in the lobby network.

Reply:
There's no reason to log MOTD. It's meant to be something that is just around for your current connection. On reconnect, you'll get the new MOTD, so the only time you won't have the MOTD is when you are currently disconnected from the IRC server.

Reply:
I guess I'm expecting servers to update their MOTDs frequently, which is definitely not extremely usual. But for example, I like being able to refer to older Freenode MOTDs, as they display news / info in there (for example about their conference), but it might be too niche and/or not worth it.

Reply:
I feel like the advantages of this for a few people would be outweighed by the confusion of having multiple motds appearing when you start the lounge, personally. But I don't feel about it strongly.

Reply:
Yeah that's a good conclusion. I do see benefits in keeping track of older MOTDs, but you're right that the benefits do not outweigh the drawbacks. Another thing that I'm happy to see what feedbacks we get, i.e. if we ever get users confused by this or not, and we can then rethink this.

Reply:
Yeah, if people find it confusing, happy to revisit.